imports/client/theme.ts

```diff
@@ -0,0 +1,372 @@
+import type { ThemeConfig } from "react-select";
+
+export type GuessState = keyof Theme["colors"]["guess"];
+export type SolvednessState = keyof Theme["colors"]["solvedness"];
+
+export interface Theme {
+  basicMode: string;
+  colors: {
+    background: string;
+    text: string;
+    primary: string;
+    secondary: string;
+    muted: string;
+    danger: string;
+    warning: string;
+    info: string;
+    success: string;
+    callStateIcon: string;
+    mutedIconBorder: string;
+    deafenedIconBorder: string;
+    mutedIconText: string;
+    deafenedIconText: string;
+    remoteMuteButtonHoverBackground: string;
+    remoteMuteButtonHoverText: string;
+    chatterSectionBackground: string;
+    firehosePreWrapBackground: string;
+    firehosePreWrapText: string;
+    codeBlockBackground: string;
+    codeBlockText: string;
+    mentionSpanBackground: string;
+    mentionSpanText: string;
+    selectedMentionSpanShadow: string;
+    selectedMentionSpanBackground: string;
+    selectedMentionSpanText: string;
```

Comment:
commented in FancyEditor, but `selectedMentionSpanBackground` and `selectedMentionSpanText` appear unused